Geometry/RPCGeometryBuilder/plugins/RPCGeometryValidate.cc

```diff
@@ -0,0 +1,305 @@
+/*
+//\class RPCGeometryValidate
+
```

Comment:
@slomeo in which context is this supposed to be used? Not the DQM workflows, just for standalone validation?

Reply:
@fabiocos It is used only for standalone validation (i.e histos production)

Reply:
@slomeo so it would be better to have it in a test directory